Gaijinizer patch/Map196.txt

```diff
@@ -175,24 +184,27 @@ magic of tranformed species. It was not just
 
 \\C[10]今までの道のりで『同じ魔力を得ていた者』からな……
 > CONTEXT: Map196/events/1/pages/0/163/Dialogue/enemy17
-Hohoho...The place, \\N[1]. I borrowed the
-magic power. \\C[10]From "the person who has obtained
-the same magic" in the way up to now ...#MTLed
+Hohoho... In short, \\N[1] was borrowing his
+magic power \\C[10]from on the journey so far
+"same magic power obtained person"...#IHaveNoFuckingIdea
 > END STRING
```

Comment:
Let's try to parse this:
>_**Ohoho... In short, \\N[1] was borrowing his
magic power from someone.
\\C[10]Someone who was with him all along...**_